### PR TITLE
Data table id string

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?179685

@dannyroberts @orangejenny 

img_0780

### CODE REVIEW

**Comment by orangejenny** [comment]
looks good to me. i will be sad the day you learn to take screenshots from your phone, there's such...authenticity...in that photo.

**Comment by czue** [comment]
build fail looks real
```
======================================================================
FAIL: test_report_module (corehq.apps.app_manager.tests.test_suite.SuiteTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_suite.py", line 661, in test_report_module
    "./detail[@id='reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.summary']",
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/util.py", line 20, in assertXmlPartialEqual
    self.assertXmlEqual(expected, actual, normalize=False)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/util.py", line 26, in assertXmlEqual
    _check_shared(expected, actual, LXMLOutputChecker(), "xml")
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/util.py", line 117, in _check_shared
    raise AssertionError(message)
AssertionError: XML mismatch
--- want.xml
+++ got.xml
@@ -66,7 +66,7 @@
     <detail id="reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.data" nodeset="rows/row">
       <title>
         <text>
-          <locale id="cchq.reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.name"/>
+          <locale id="cchq.report_data_table"/>
         </text>
       </title>
       <field>
```

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -3447,7 +3447,7 @@ def _column_to_field(column):
             id='reports.{}.data'.format(self.uuid),
             nodeset='rows/row',
             title=suite_xml.Text(
-                locale=suite_xml.Locale(id=id_strings.report_name(self.uuid)),
```
Comment (czue) [inline corehq/apps/app_manager/models.py]: can `report_name` be deleted or is that still used somewhere else?

Comment (nickpell) [inline corehq/apps/app_manager/models.py]: used here: https://github.com/dimagi/commcare-hq/pull/8230#discussion-diff-39341843R3393

**Comment by nickpell** [comment]
haha @orangejenny the actual problem is that I've changed my password since using that phone regularly, don't have my password manager on that phone, and need my password to login to google play to download the password manager.  Also the web version of the password manager doesn't work on mobile.  so until I solve that conundrum ... enjoy

**Pull request merged**